src/array_api_stubs/_draft/__init__.py

```diff
@@ -21,4 +21,6 @@
 __array_api_version__: str = "YYYY.MM"
 """
 String representing the version of the array API specification which the conforming implementation adheres to.
+
+.. versionadded:: 2022.12
```

Comment:
```suggestion

.. versionadded:: 2022.12
```

I think this should just be removed, as this was part of the API in `2021.12`, just it wasn't in the stubs.

Reply:
I'm not sure if the goal is to have a record long term of what APIs where changed, in case it is v202x and you want to know when the API appeared or was modified.

Reply:
@steff456 I think that was @honno's point - it's _not_ new in 2022.12 - see https://data-apis.org/array-api/2021.12/search.html?q=__array_api_version__

Reply:
Sorry!!! Yes you are right!